src/main/build/debug.c

```diff
@@ -120,5 +120,6 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     "TPA",
     "S_TERM",
     "SPA",
+    "ALTHOLD",
     "TASK",
```

Comment:
Needs to be at the end.
```suggestion
    "TASK",
    "ALTHOLD",
```

Reply:
(whole list may be refactored using designated initializers - then it's a bit harder to get it wrong:)
```suggestion
    [DEBUG_TASK] = "TASK",
    [DEBUG_ALTHOLD] = "ALTHOLD",
```

Reply:
Yes that reminds me :) - we need to that in a separate PR

Reply:
OK sorry I thought 'task' was 'special' and had two be at the end :-)